bin/underreact.js

```diff
@@ -2,17 +2,12 @@
 'use strict';
 
 const yargs = require('yargs');
-const path = require('path');
-const chalk = require('chalk');
-const webpack = require('webpack');
 
 const start = require('../commands/start');
 const build = require('../commands/build');
 const serveStatic = require('../commands/serve-static');
 const logger = require('../lib/logger');
-const { DEFAULT_CONFIG_NAME } = require('../lib/constants');
-const createConfig = require('../lib/config');
-const getDefaultConfig = require('../lib/default-underreact.config');
+const config = require('../lib/config');
```

Comment:
Is this supposed to point to `createUrc`?

Reply:
No, to fully test the entire thing (https://github.com/mapbox/underreact/pull/14/files#diff-76c595bf9e5f6dac835e81cd694d425eR8) I extracted the code in `bin/underreact` to `/config`.